examples/resources/ovhcloud_connect_config_pop/resource.tf

```diff
@@ -0,0 +1,12 @@
+data "ovh_ovhcloud_connect" "occ" {
+   service_name = "XXX"
+}
+
+resource "ovh_ovhcloud_connect_pop_config" "pop" {
+    service_name = data.ovh_ovhcloud_connect.occ.service_name
+    interface_id = tolist(data.ovh_ovhcloud_connect.occ.interface_list)[0]
+    type = "l3"
+    customer_bgp_area = AAA
```

Comment:
you should use a fake AS number here to reflect the real type of the parameter (same for `ovh_bgp_area`)